services/service-core/src/config/config.validator.ts

```diff
@@ -0,0 +1,47 @@
+import * as Joi from 'joi';
+
+export interface IKeycloakConfiguration {
```

Comment:
Do not prefix interfaces with `I`. Homework: do some research online for reasons why. Let's discuss more after you get some insights

Reply:
from all i found this 2 make sense
1, Readability and Expressiveness: Some argue that a well-named interface should be clear and expressive on its own without needing a prefix. If the interface name is appropriately descriptive, the 'I' might be redundant.

2, Unnecessarily Revealing Implementation Details: The 'I' prefix might expose the underlying implementation details, which could be considered a violation of the abstraction principle. Interfaces should define a contract without revealing how it will be implemented.